lib/api/core/plugins/pluginsManager.js

```diff
@@ -1062,6 +1064,30 @@ class PluginsManager {
             : {}
         };
 
+      // load customs errors configuration file
+      if (plugin.manifest.raw.errors) {
+        try {
+          const pluginCode = this.config[plugin.manifest.name] && this.config[plugin.manifest.name]._pluginCode
+            ? this.config[plugin.manifest.name]._pluginCode
+            : 0x00;
+          loadPluginsErrors(plugin.manifest.raw, pluginCode);
+          this.kuzzle.log.info(`[${plugin.manifest.name}] Custom errors successfully loaded.`);
+        } catch (err) {
+          if (err.message.match(/Error configuration file/i)
```

Comment:
Why matching with a regexp ? You can throw custom errors with `assert`
```
assert(/* something */, errorsManager.getError(/* ... */));
```

Reply:
even if i use assert, i'll need regexp to match the error message

Reply:
Not if you use an error from the errorManager, so you can use `error.errorName`
But I won't block this PR for this